locale/en_US/mozfest_session_proposal_2018.json

```diff
@@ -0,0 +1,12 @@
+{
+  "email_subject": "Your MozFest session proposal",
+  "hi": "Hi {{first_name}},",
+  "thank_you": "Thank you for submitting a session proposal for MozFest!",
+  "session_curation_process": "All proposals will be reviewed and curated on GitHub in the <a href=\"{{ github_program_repo_url }}\">MozFest 2018 program repo</a>. Curation will begin in August when the Call for Proposals closes. In early September, you’ll be notified whether your session has been accepted or not.",
```

Comment:
I have a question about `github_program_repo_url`: Is there an `event.data.github_program_repo_url` I can use in the worker?

Reply:
@patjouk did you mean a default value for `github_program_repo_url` or?

Reply:
I'm more curious on where the value for `github_program_repo_url` is defined. I plan to add it in the worker [like this](https://github.com/mozilla/sawmill/blob/master/worker/mozfest_session_proposal_2017.js#L10-L13). I just want to be sure I'm not passing an empty value by accident :)

Reply:
Did you mean this? https://github.com/mozilla/mozillafestival.org/blob/master/lib/proposal-handler.js#L177-L200 (this is last year's code still)

Reply:
Yup, that's what I meant. Thanks!